src/ga4gh/vrs/extras/translator.py

```diff
@@ -56,24 +62,28 @@ def __init__(self,
         self.hgvs_tools = None
 
 
-    def translate_from(self, var, fmt=None):
+    def translate_from(self, var, fmt=None, require_validation=True):
         """Translate variation `var` to VRS object
 
         If `fmt` is None, guess the appropriate format and return the variant.
         If `fmt` is specified, try only that format.
+        If `require_validation` is `True` then validation checks must pass in order to
+            return a VRS object. A `ValidationError` will be raised if validation checks
+            fail. If `False` then VRS object will be returned even if validation checks
+            fail.
 
         See also notes about `from_` and `to_` methods.
         """
 
         if fmt:
             t = self.from_translators[fmt]
-            o = t(self, var)
+            o = t(self, var, require_validation=require_validation)
```

Comment:
The dispatch table approach is pretty clean when each translator has the same arguments, but if we need different params for different translators, I think we probably have to call them within some if/then blocks. A little inelegant but IMO cleaner than passing a bunch of unused arguments to every other function.

Reply:
I agree. I believe we will be adding validation checks to other methods as well (just haven't gotten around to it). Can make a separate issue for this